libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -944,14 +965,17 @@ static CK_RV prvRsaKeyAttParse( const CK_ATTRIBUTE * pxAttribute,
                 }
                 else
                 {
-                    LogError( ( "Expected a public key for CKA_VERIFY or CKA_EC_POINT." ) );
+                    LogError( ( "Failed parsing EC key template. The key type"    \
```

Comment:
The backslashes can be removed in the logging calls. Also, spaces between the line broken words should be added